### ISSUE TITLE
unknown URI: faust://document/wa/2_III_H.15II

### ISSUE THREAD

**Issue by sandrakrause** [faustedition/xml#610]
Erich Schmidt unterscheidet bei H.15 und H.15II zwischen Vorder- und Rückseite einer Handschrift. (Zwei Versionen eines Textes)
Wie soll verfahren werden?
Wenn diese Unterscheidung beibehalten werden soll, müsste man sich hier was überlegen (als Inscription behandeln, ...)

**Comment by dietmarpravida** [comment]
Da identische Verse vorkommen, die man gerne chronologisch sortieren würde (15 vor 15II), ist das ein relevantes Problem.

**Comment by gerritbruening** [comment]
> Da identische Verse vorkommen, die man gerne chronologisch sortieren würde

Das ist bei Inskriptionen ja öfters so. Für mich ein klarer Fall von Inskription, sollte so behandelt werden wie die übrigen.

**Comment by dietmarpravida** [comment]
d.h. wir werten diese Unterscheidung nicht aus, H.15 wird zeitlich mit H.15II eingeordnet (wie sie gereiht werden, ist dann ein anderes Problem, oder?)

**Comment by gerritbruening** [comment]
> H.15II

ist entweder subidno oder Inskription, können wir uns nach Opportunität aussuchen.

**Comment by sandrakrause** [comment]
Habe H.15II jetzt mal als subidno in den Metadaten zu H.15 ergänzt.
Kann man ja wieder ändern, sollte man sich für etwas anderes entscheiden.

**Comment by thvitt** [comment]
hm, ich mach noch gar nix mit den subidnos. Soll ich die auch zum Resolven der Makrogenese-URIs verwenden?

**Comment by gerritbruening** [comment]
Ja, zum resolven auf jeden Fall. Kann halt  sein, dass dann die Kante H.15 --> H.15 rauskommt und ignoriert werden müsste.

**Comment by thvitt** [comment]
solche loops fliegen eh raus

**Comment by sandrakrause** [comment]
Ich hab bei anderen Fällen jetzt auch teilweise subidnos verwendet, um sie zu definieren.
Wäre das denn ein großes Problem, die noch zu verwenden?
Wie könnte man das sonst stattdessen lösen?

**Comment by dietmarpravida** [comment]
M.E. solten bei Inskriptionen Inskriptionen kodiert werden, nicht subidnos (deren Verwendung mir nicht so kar ist, aber vielleicht liegt das an mir)

**Comment by gerritbruening** [comment]
> M.E. solten bei Inskriptionen Inskriptionen kodiert werden

finde ich grundsätzlich richtig, allerdings kann es den Bedarf geben, bestimmte Siglen als Siglen aufznehmen, einfach weil sie äußerlich als Siglen in der Welt sind (und nicht als Inskriptionsbezeichner). Dass diese außerdem im Grunde Inskriptionen bezeichnen, muss dann unabhängig und zusätzlich kodiert werden.
